eng/Badge.proj

```diff
@@ -9,7 +9,7 @@
     <PropertyGroup>
       <!-- Replace '-' with '_' for os names like 'linux-musl' -->
       <VersionBadgeMoniker>$(OSName.Replace('-', '_'))_$(TargetArchitecture)</VersionBadgeMoniker>
-      <VersionBadgeMoniker Condition="'$(IsLinuxPortable)' == 'true'">linux_$(TargetArchitecture)</VersionBadgeMoniker>
+      <VersionBadgeMoniker Condition="'$(IsLinuxglibc)' == 'true'">linux_$(TargetArchitecture)</VersionBadgeMoniker>
```

Comment:
I bet this dates back to when we had non-portable Linux builds. If OSName is "linux" here (which I think it will be), we could remove this check entirely.